ca/certificate-authority.go

```diff
@@ -596,7 +596,17 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	}
 
 	// Submit the certificate to any configured CT logs
-	go ca.Publisher.SubmitToCT(certDER)
+	go func() {
```

Comment:
Right, this was meant to catch RPC errors. Hmm, any way to distinguish a RPC comms error from a remote logic error?